pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
   "jmespath>=0.6.2",
   "jsonschema>=4.0.1",
   "numpy>=1.18",
+  'numpy<1.25,>=1.18; python_version < "3.9"',
```

Comment:
Oldest supported numpy version is 1.20 (and its support ends January 31). We should bump the minimum numpy soon.

Reply:
Do you want that change included in this PR?

Reply:
No

Reply:
What other changes are required to merge this PR? It's currently showing that you have 'changes requested'.